WalletWasabi.Gui/ViewModels/WasabiDocumentTabViewModel.cs

```diff
@@ -71,13 +75,23 @@ public virtual void OnDeselected()
 			IsSelected = false;
 		}
 
-		public virtual void OnOpen()
+		void IDockableViewModel.OnOpen()
```

Comment:
Add access modifier.

Reply:
This method signature is a special case, its an explicit interface implementation, so you are not allowed to put an access modifier there. (Notice the 'IDockableViewModel.' part).

The reason for this, is the public method is now:

`public virtual void OnOpen(CompositeDisposable disposables)`

we dont want to implement that `OnOpen(void)` method publically, as the end user mustn't use this anymore. Implementing it explicitly like this means its hidden and only visible if they cast the object to `IDockableViewModel`.

Reply:
Thank you for the explanation.